blog/.vuepress/plugins/index.js

```diff
@@ -1,22 +1,23 @@
-const { find, build, extract, tags } = require("./pages");
```

Comment:
I removed what I think are unecessary files since we only have 3 modules to creates the pages and sidebars.